custom_components/tuya_local/devices/kogan_lx8_vacuum.yaml

```diff
@@ -0,0 +1,323 @@
+name: Kogan LX8 Robot Vacuum
+products:
+  - id: 1bgkj0zbhbw7fppy
+    name: Kogan LX8 Robot Vacuum
+primary_entity:
+  entity: vacuum
+  dps:
+    - id: 101
+      name: activate # clean_switch
+      type: boolean
+    - id: 102
+      name: pause # pause_switch
+      type: boolean
+    - id: 104
+      name: command # clean_mode
+      type: string
+      optional: true
+      mapping:
+        - value: pause
```

Comment:
This does not seem right.  What are you trying to do here, and why did you not just copy the config from LX10 for this dp?

Reply:
Have copied the lx10 now